src/content/docs/docs/integrations/google-genai.mdx

```diff
@@ -16,10 +16,6 @@ import LanguageContent from '../../../../components/LanguageContent.astro';
 
 The Google GenAI plugin provides a unified interface to connect with Google's generative AI models through the **Gemini Developer API** using API key authentication. It is a replacement for the previous `googleAI` plugin.
 
-:::tip[Enterprise Users]
-For enterprise features like grounding, context caching, Vector Search, and Model Garden access, see the [Vertex AI plugin](/docs/integrations/vertex-ai).
```

Comment:
The vector search plugin and Model Garden plugin are still in the old vertexai package... Not to be confused with the new vertexai Plugin from the google-genai package...